interpreter.py

```diff
@@ -2,8 +2,9 @@
 from rpython.rlib import jit
 from type_wrapper import String, Integer, Float, Ptr, List,\
                          Value, NoValue, NumericValue, BasicBlock
-from llvm_wrapper import *
+from llvm_objects import W_Module
 from state import State
+from llvm_wrapper import *
```

Comment:
"explicit is better than implicit". Anyway, most of the functions in ``llvm_wrapper`` are no longer used in interpreter.py, because they are hidden in ``llvm_objects``.